src/test/resources/json/get_consignment_for_export.json

```diff
@@ -40,6 +40,10 @@
             ],
             "containerSignatureFileVersion": "DROID_SignatureFile_V96.xml",
             "binarySignatureFileVersion": "container-signature-20200121.xml"
+          },
+          "antivirusMetadata": {
+            "software": "software",
+            "softwareVersion": "version"
```

Comment:
Might be worth changing the value to `"softwareVersion"` for consistency?

Reply:
Yep, makes sense, I've changed it.